src/node/rpc/test/frontend_test.cpp

```diff
@@ -208,15 +208,15 @@ class TestAlternativeHandlerTypes : public BaseTestFrontend
   {
     open();
 
-    auto command = [this](auto& args) {
-      args.rpc_ctx->set_response_status(HTTP_STATUS_OK);
+    auto command = [this](auto& ctx) {
+      ctx.rpc_ctx->set_response_status(HTTP_STATUS_OK);
```

Comment:
I wonder to what extent splitting `ctx.rpc_ctx` into `ctx.request` and `ctx.response` would be justifiable.

Reply:
Yeah, we should do this. It was slightly awkward in the original HTTP migration where there was some shared state (eg - the packing, whether that was JSON or msgpack, being the same for both request and response). I believe these are now cleanly separated - one may inform the other but they're tracked in distinct fields. This should also make `reset_response` simpler + safer.

Reply:
https://github.com/microsoft/CCF/issues/2568